data/functions.sql

```diff
@@ -1006,28 +1006,30 @@ BEGIN
     RETURN '{}'::jsonb;
   END IF;
 
-  -- first, try the countries table
-  SELECT
-    min_label, max_label INTO min_zoom, max_zoom
-    FROM ne_10m_admin_0_countries c
-    WHERE c.wikidataid = wikidata_id;
+  -- if it's a country, only look it up in the iso and tlc tables
+  IF place_tag='country' OR place_tag='unrecognized' THEN
+      SELECT
+        min_label, max_label INTO min_zoom, max_zoom
+        FROM ne_10m_admin_0_countries_iso i
+        WHERE i.wikidataid = wikidata_id;
 
-  -- if that fails, try map_units (which contains some sub-country but super-
-  -- state level stuff such as England, Scotland and Wales).
-  IF NOT FOUND THEN
-    SELECT
-      min_label, max_label INTO min_zoom, max_zoom
-      FROM ne_10m_admin_0_map_units mu
-      WHERE mu.wikidataid = wikidata_id;
+      IF NOT FOUND THEN
+        SELECT
+            min_label, max_label INTO min_zoom, max_zoom
+        FROM ne_10m_admin_0_countries_tlc t
+        WHERE t.wikidataid = wikidata_id;
+      END IF;
+
+      IF NOT FOUND THEN
+        RETURN '{}'::jsonb;
+      END IF;
   END IF;
 
-  -- try states and provinces
-  IF NOT FOUND THEN
```

Comment:
Don't you want to keep this in here to bail early if found?

Reply:
You're right.  I thought I was returning on green line 1024, but that's only if we didn't find it.  Updated to check if it's a country else do something else, then return once at the end.

Reply:
Fixed in 31a5b4a27b76b89b24214157b4758b3ddebad030